run/openssl2john.py

```diff
@@ -65,15 +65,16 @@ def process(filename, plaintext=None, cipher=0, md=0):
                 rdata = rdata.decode("ascii")
             sys.stdout.write("%s:$openssl$%s$%s$8$%s$%s$0$%s$%s$%s\n" %
                              (filename, cipher, md, salt, last_chunk,
-                              len(rdata), rdata, s))
+                              len(rdata) // 2, rdata, s))
```

Comment:
Oops, you caught me there 😆  Yeah I usually skim through it more or less quickly. I was mostly wondering about the `// 2` syntax. What's that? A plain division? Why two of them?